fe/src/pages/article-list.page.tsx

```diff
@@ -0,0 +1,17 @@
+import { SearchArticleList } from "./search.article-list.page";
+import { PageProps } from "../utils/page-wrapper";
+import { GeneralArticleList } from "./general.article-list.page";
+import { PeopleArticleList } from "./people.article-list.page";
+
+export function ArticleListPage({ routerInfo }: PageProps) {
+  if (routerInfo.locationId != null) {
+    return <GeneralArticleList routerInfo={routerInfo}></GeneralArticleList>;
+  }
+  if (routerInfo.people) {
+    return <PeopleArticleList routerInfo={routerInfo} />;
+  }
+  if (routerInfo.search) {
+    return <SearchArticleList />;
+  }
+  return <div>Something went worong.</div>;
```

Comment:
Agreed. Actually the "404" page, which is for a "catches error" in our logic. It should be a quick modification, I guess.

@lidachao111222, see if you can quickly do it. If not, I guess this is not our main focus for this PR, we should quickly move on and do it in another PR.

Reply:
"If we consider it another way, this could also make sense. It means no string is used for the query, if we stay on the search page, then user can actually just put another query string to continue. So that user is not required to put something to search at the first time. Instead, the can use the search bars on other pages as a navigation to the search page."

yes

Reply:
so what's the action items here? are you gonna update the logic here.

Btw you spelled wrong wrong.

Reply:
routerInfo added.